### PR TITLE
Move more targets to build.d

### PR DESCRIPTION
**Pull request by marler8997**

### CODE REVIEW

**Review by wilzbach — CHANGES_REQUESTED** [review]
Hmm, thanks a lot for your continued support and work on the build script, but in this case I think that these targets should just be removed. These are all legacy relicts that are/were solely used by Walter that predate git. Git does line conversions automatically on Windows...

SCP and ZIP are dead as well, btw.

**Comment by marler8997** [comment]
@wilzbach yeah I got that idea when I realized that `tolf` removes all the trailing newlines, so it's obviously not used anymore.  I'd like it if someone more knowledgeable of what' supposed to be supported could go through the Makefiles and just remove what's no longer used.  We might need @WalterBright to do that.

**Comment by marler8997** [comment]
Re-opening since @WalterBright has indicated he still uses these targets https://github.com/dlang/dmd/pull/10551#issuecomment-552167355

File: src/build.d [hunk 1]
```diff
@@ -5,6 +5,12 @@ DMD builder
 Usage:
   ./build.d dmd
 
+detab, tolf, install targets - require the D Language Tools (detab.exe, tolf.exe)
+  https://github.com/dlang/tools.
+
+zip target - requires Info-ZIP or equivalent (zip32.exe)
+  http://www.info-zip.org/Zip.html#Downloads
```
Comment (PetarKirov) [inline src/build.d]: Remove this paragraph as this target is not defined by `dmd`.

File: src/posix.mak [hunk 2]
```diff
@@ -463,7 +463,7 @@ cxx-unittest: $(GENERATED)/build
 
 ######################################################
 
-zip: $(GENERATED)/build
+zip: detab tolf $(GENERATED)/build
```
Comment (PetarKirov) [inline src/posix.mak]:
```suggestion
zip: $(GENERATED)/build
```
`detab` and `tolf` are not defined in this makefile. Of course, would be better if we delete the `zip` target altogether, but let's leave this for another PR.

**Review by PetarKirov — APPROVED** [review]

File: src/build.d (hunk 1 above)
Comment (marler8997) [reply src/build.d]: I added the zip target recently.

File: src/posix.mak (hunk 2 above)
Comment (marler8997) [reply src/posix.mak]: thanks for finding that

File: src/build.d (hunk 1 above)
Comment (PetarKirov) [reply src/build.d]: Sorry for the noise - I missed that.

**Review by PetarKirov — APPROVED** [review]
